assets/sass/05-blocks/columns/_editor.scss

```diff
@@ -0,0 +1,6 @@
+.wp-block-columns .wp-block,
+.wp-block-columns .wp-block-column {
+	// Allow the width of a block that lives inside the columns block to be
+	// determined by the parent columns block
+	max-width: none;
```

Comment:
I'm not sure about this. Should the width of a block inside a columns block be unconstrained? 

It overrides this rule below which is quite broad — limiting the width of any block (included nested blocks) in the editor to the default width, unless it has a wide or full alignment:

https://github.com/WordPress/twentytwentyone/blob/9e4f5213502ad6007c2c5267b8ad08ed4c5e53d7/assets/sass/05-blocks/utilities/_editor.scss#L139-L143

Reply:
We might want to see if anything happens with https://github.com/WordPress/gutenberg/issues/24055 or https://github.com/WordPress/gutenberg/issues/25655. Alternately, @mtias, do you have any thoughts here?

Reply:
Yeah, I feel like we should wait for a natively-supported solution to this problem. Otherwise when folks switch themes, their text will end up looking really narrow all of a sudden, while the rest of the page stays the same.

Reply:
Agreed on not coding around this issue and to improve the affordances we offer in Gutenberg as a whole. I'm also curious to know when you would use a single column block vs a group block set to wide?

Reply:
Can't we make it a block style for a heading?

Reply:
I think the only use case for a single column (versus a Group block) would be to get the text to expand the full-width. But that's just a hack, really. I think if Gutenberg were to allow you to set a headings block to wide width, that would be the cleanest, simplest solution for now.

Reply:
FWIW, I don't think that #25857 would solve this for us either, since we'd still need to override the default max-width imposed on all blocks.